collector/default.go

```diff
@@ -6,7 +6,7 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/cespare/xxhash"
```

Comment:
@mheese this might break claims reporting. Backend expects it to be v1. https://github.com/aporeto-inc/underwater/blob/968acc16112498a59065bd4b1f95974cb2fc0e42/core/policy/services/claims.go#L30

Reply:
ah ... ok, good point 👍  ... but we didn't have a constraint on this before in trireme (and only came in through the enforcer of course)

Reply:
@sibicramesh reverted